book/website/reproducible-research/licensing/licensing-ml.md

```diff
@@ -48,6 +72,66 @@ alt: An illustration depicting a flow chart diagram for a decision making proces
 The OpenRAIL flow chart aids the selection and naming of a license for an ML project. Danish Contractor, Carlos Muñoz Ferrandis, Jenny Lee, & Daniel Mcduff. (2022, August).
 ```
 
+(rr-licensing-ethics-copyright-responsible-rail)=
+### OpenRAIL License types
+
+These same principles developed in 'ethical source' apply to the 'Open' variants of the licences from RAIL (Responsible AI Licences).
+In that, they are attempting to place restrictions on the uses to which licensees can put the thing being licenced.
+Traditional software has many of the same concerns which affect machine learning models, and indeed also often contain assets such as images which may be licenced differently from software with which they are bundled.
+The primary differences being governance of data used in training the models (see: Data Governance for the Machine Learning Pipeline) and the lack of interpretability of the decisions of many ML systems, though this latter point can also be an issue for conventional systems if they are closed.
```

Comment:
Should there be a link here?

Reply:
addressed in [556c4bc](https://github.com/the-turing-way/the-turing-way/pull/3142/commits/556c4bca6abf3f40d31967ed21996ddb083e94fc)